Core/UI/HomeView.cpp

```diff
@@ -40,82 +40,41 @@ HomeView::HomeView(lv_obj_t* parent, HomeViewModel& viewModel) : View(parent, vi
     lv_chart_set_type(lapTimeBarGraph, LV_CHART_TYPE_BAR);
     lv_obj_set_size(lapTimeBarGraph, 200, 150);
     lv_obj_center(lapTimeBarGraph);
-    lv_chart_set_range(lapTimeBarGraph, LV_CHART_AXIS_PRIMARY_Y, 0, 100);
```

Comment:
Still need to play around with this, the values don't fit in the chart at the moment. Easy fix, but will do in another commit.